client/components/main/layouts.jade

```diff
@@ -13,6 +13,13 @@ template(name="userFormsLayout")
     h1.at-form-landing-logo
       img(src="{{pathFor '/wekan-logo.png'}}" alt="Wekan")
     +Template.dynamic(template=content)
+    div.at-form-lang
+      select.select-lang.js-userform-set-language
```

Comment:
If put it into registration form, maybe it should also apply to enroll form when user set initial password ... 

I am not very sure whether it's needed or not, as the default language of user form is already user navigator language. 

I just thought of providing a way for user to change the language of the user form in any state, in case it's not preferred.